plugins/mcreator-localization/help/default/damagetype/player_death_message.md

```diff
@@ -0,0 +1,5 @@
+This message is displayed in chat when a player dies from damage of this type, but was recently hurt by another entity.

```

Comment:
I would reword

> but was recently hurt by another entity.

to

> but the damage itself originates from another entity

Reply:
This is used for example when a player was attacked by a zombie, but dies in lava immediately after, so that's not entirely correct

Reply:
Ok. I did not know this such damage is also tracked, nice. Then it is fine :)